### PR TITLE
docs(toh-4/dart): copyedits

### PR DESCRIPTION
**Pull request by chalin**
Contributing to #1628.

### CODE REVIEW

File: public/docs/dart/latest/tutorial/toh-pt4.jade [hunk 1]
```diff
@@ -59,20 +60,20 @@ code-example(language="bash").
...
+  We can refactor this hero data acquisition business to a single service that provides heroes, and
   share that service with all components that need heroes.
 
   ### Create the HeroService
   Create a file in the `lib` folder called `hero_service.dart`.
 .l-sub-section
   :marked
     We've adopted a convention in which we spell the name of a service in lowercase followed by `_service`.
-    If the service name were multi-word, we'd spell the base filename with lower underscore case (AKA "snake_case").
+    If the service name were multi-word, we'd spell the base filename in lower underscore case (also called [snake_case](../guide/glossary.html#!#snake_case)).
```
Comment (kwalrath) [inline public/docs/dart/latest/tutorial/toh-pt4.jade]: Fix the snake_case link? (It just went to the top of the glossary page, for me.)

**Comment by kwalrath** [comment]
LGTM, though we should fix that snake_case link.

File: public/docs/dart/latest/tutorial/toh-pt4.jade (hunk 1 above)
Comment (chalin) [inline public/docs/dart/latest/tutorial/toh-pt4.jade]: The `snake_case` link when in with the toh-3 changes ... Ah, I know what happened. Let me fix this.

**Comment by chalin** [comment]
The `snake_case` entry went in with the toh-3 changes. You need to rebuild the docs to get it to be picked up. I just did that and it works -- see below.
screen shot 2016-06-15 at 2 15 39 pm

**Comment by kwalrath** [comment]
I can't get the link to work in my build, but then building anything more than the devguide fails for me right now. I'll take your word that it works.

**Pull request merged**